Server/src/routes/poi.route.ts

```diff
@@ -196,18 +197,23 @@ export class PoiRoute {
 		}
 
 		const track = (await database.tracks.getById(userData.trackId)) ?? undefined
+		if (track == undefined) {
```

Comment:
This should then probably be changed to a comparison to `null`, although `(...) == null` and `(...) == undefined` are semantically identical.